packages/ts/lit-form/test/Validation.test.ts

```diff
@@ -754,6 +754,32 @@ describe('@vaadin/hilla-lit-form', () => {
         }
       });
 
+      it('should display server validation error for product description', async () => {
```

Comment:
Where in the test the changes made are tested? 🤔 Maybe, it would be better to add a check for transformed message?

Reply:
Line 769 contains `'products[0].description'` in square bracket notation and the test verifies that the associated error is propagated to the binder. And I verified that the test fails without the code change.

Reply:
Ah, ok, I see. Thanks for clarification